ksql-functional-tests/src/test/resources/query-validation-tests/array-functions.json

```diff
@@ -0,0 +1,82 @@
+{
+  "comments": [
+    "Tests covering the use of the ROUND(DOUBLE), FLOOR(DOUBLE), CEIL(DOUBLE) functions."
+  ],
+  "tests": [
+    {
+      "name": "calculate ROUND function",
+      "statements": [
+        "CREATE STREAM test (v DOUBLE) WITH (kafka_topic='test_topic', value_format='JSON');",
+        "CREATE STREAM OUTPUT AS SELECT ROUND(v) as R0, ROUND(v, 0) as R00, ROUND(v, 1) as R1, ROUND(v, 2) as R2, ROUND(v, 10) as R10 FROM test;"
```

Comment:
The implementation you've added supports negative scales e.g. `ROUND(111, -1)` -> `110`.

Can we add a test case / column for this please?

Reply:
Can you also add a test case/column where the second param to `ROUND` is `null`.  You can do this explicitly in the statement, or have the second param also come from an input field:

```
CREATE STREAM test (v DOUBLE, scale INT) WITH (kafka_topic='test_topic', value_format='JSON');
CREATE STREAM OUTPUT AS SELECT ROUND(v, scale) FROM test;
```

Then have inputs:
```
"inputs": [
        {"topic": "test_topic", "value": {"v" : 1.2, "scale": 0}},
        {"topic": "test_topic", "value": {"v" : 1.7, "scale": -1}},
        {"topic": "test_topic", "value": {"v" : 1.7, "scale": null}},
```